packages/server/modules/webhooks/graph/resolvers/webhooks.js

```diff
@@ -2,14 +2,15 @@ const { ForbiddenError } = require('apollo-server-express')
 
 const { authorizeResolver } = require('@/modules/shared')
 const {
-  getWebhook,
   updateWebhook,
   deleteWebhook,
   getStreamWebhooks,
   getLastWebhookEvents,
   getWebhookEventsCount
 } = require('../../services/webhooks')
 const { Roles } = require('@speckle/shared')
+const { getWebhookByIdFactory } = require('../../repositories/webhooks')
```

Comment:
no relative imports

Reply:
I will move that file to TS in the end and ESM
so for now I am not changing those, I will  change them all at once when I will move the file.
There are also many imports already in relative form